README.md

```diff
@@ -18,7 +18,7 @@ In your Flutter project, add the package to your dependencies
 
 ```yml
 dependencies:
-    reorderable_carousel: ^0.3.1
+    reorderable_carousel: ^0.4.0
```

Comment:
I have some other things I want to do before rolling out this release, so I'll handle all the version stuff
```suggestion
    reorderable_carousel: ^0.3.1
```